iphone/Classes/TiUIListView.m

```diff
@@ -2225,22 +2192,22 @@ -(CGFloat)contentHeightForWidth:(CGFloat)width
 -(void)clearSearchController:(id)sender
 {
     if (sender == self) {
-        RELEASE_TO_NIL(tableController);
-        RELEASE_TO_NIL(searchController);
         [searchViewProxy ensureSearchBarHeirarchy];
     }
 }
 
 -(void)initSearchController:(id)sender
 {
-    if (sender == self && tableController == nil) {
-        tableController = [[UITableViewController alloc] init];
-        [TiUtils configureController:tableController withObject:nil];
-        tableController.tableView = [self tableView];
-        searchController = [[UISearchDisplayController alloc] initWithSearchBar:[searchViewProxy searchBar] contentsController:tableController];
-        searchController.searchResultsDataSource = self;
-        searchController.searchResultsDelegate = self;
+    if (sender == self && searchController == nil)
+    {
+        searchController = [[UISearchController alloc] initWithSearchResultsController:nil];
```

Comment:
Why are you initializing it with `nil`? I guess we should rather manage an own table-view for it, maybe even in an own helper-class.